### ISSUE TITLE
Council Guide and AC reviews

### ISSUE THREAD

**Issue by chrisn** [w3c/Guide#176]
The [Council Guide](https://www.w3.org/Guide/council/council.html) says "groups work hard to understand different points of view" and "When that is working well, Formal Objections are rare, indeed". This gererally applies to decisions made in WGs, but not to AC charter reviews - where there is no real discussion among the AC and the [only way to express dissent is via Formal Objection](https://www.w3.org/Consortium/Process/Drafts/#ACReviewStart), so this should be reflected in the Guide.

Similarly, "implies that before an objection gets a senior review there already has been a great deal of work on the issue" may apply to WG decisions but not AC charter reviews.

"The objector, in trying to reverse the decision, has generally pulled together their best arguments and brought them to the group." As we have seen with some AC charter reviews, these can be very short, and there's no requirement for AC reviewers to bring their comments to the WG.

"There is generally clarity on what the W3C decision was" - in Process 2023, formally, a [W3C Decision](https://www.w3.org/Consortium/Process/Drafts/#def-w3c-decision) follows an AC review, so I suggest amending this text to clarify the different kinds of decisions.

Finally, a minor point, but I would prefer to avoid the word "senior", the groups involved (a WG, the AC, the AB, TAG, Team) each have different roles, but one isn't necessarily senior to another.

**Comment by ylafon** [comment]
I agree that often, the discussion around charters happens with Formal Objection, however, there is the advance notice system that gives pointers to start a discussion, usually on the charter GH repository.

The current Council Guide text says

> Similarly, most Working Group Charters are openly developed in Github. Advanced Notices inform the community when there is a new Charter to look at, the Charter development process is open for people to raise issues and there is a fair effort to resolve issues prior to someone raising a Formal Objection.

So I wonder if the Guide should amended, or the way communication around advance notices / pre-vote review should be better.

Re: `senior`: would `deeper review` be better?
